Formula/openliberty-jakartaee8.rb

```diff
@@ -0,0 +1,46 @@
+class OpenlibertyJakartaee8 < Formula
+  desc "Lightweight open framework for Java (Jakarta EE 8)"
+  homepage "https://openliberty.io"
+  url "https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/release/2021-06-29_1900/openliberty-javaee8-21.0.0.7.zip"
+  sha256 "fe8db0f5ef34aebc81b0af8e3e511672b02c9efe51f0a40c3675b28752d280be"
+  license "EPL-1.0"
+
+  livecheck do
+    url "https://openliberty.io/api/builds/data"
+    regex(/openliberty[._-]v?(\d+(?:\.\d+)+)\.zip/i)
+  end
+
+  depends_on "openjdk"
+
+  def install
+    rm_rf Dir["bin/**/*.bat"]
+
+    prefix.install %w[LICENSE NOTICES README.TXT]
+    libexec.install Dir["*"]
+    (bin/"openliberty-jakartaee8").write_env_script "#{libexec}/bin/server",
+                                                    Language::Java.overridable_java_home_env
+  end
+
+  def caveats
+    <<~EOS
+      The home of Open Liberty Jakarta EE 8 is:
+        #{opt_libexec}
```

Comment:
In what circumstances does a user need to know this path? i.e. when might they make use of this information?

Reply:
I copy it from `tomee-plume` . They are similar software.

The default `WLP_USER_DIR` in the test is `#{opt_libexec}/usr` and there are some other commands in ``#{opt_libexec}/bin`.

Reply:
Why don't we link/add env scripts for those other commands into `bin`?

Reply:
This is because these commands work only for a specific environment, and usually, we don't use them. Many command names are too common like `server` here, if we rename all, then people who are familiar with this don't know how to use it either.

Many formulas do it this way, like `tomcat`, `tomee-plums`.